### PR TITLE
update to latest version of viceroy in ci

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -9,8 +9,8 @@ defaults:
   run:
     shell: bash
 env:
-  wizer_version: 1.3.3
-  viceroy_version: 0.2.4
+  wizer_version: &wizer_version 1.3.3
+  viceroy_version: &viceroy_version 0.2.4
```
Comment (JakeChampion) [inline .github/workflows/main.yml:14]:
```suggestion
  wizer_version: 1.3.3
  viceroy_version: 0.2.4

```

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -90,10 +90,10 @@ jobs:
       matrix:
         include:
           - crate: wizer
-            version: 1.3.3 # Note: workflow-level env vars can't be used in matrix definitions
+            version: *wizer_version
```
Comment (JakeChampion) [inline .github/workflows/main.yml:93]:
```suggestion
            version: 1.3.3 # Note: workflow-level env vars can't be used in matrix definitions
```

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -90,10 +90,10 @@ jobs:
...
             options: "--all-features"
           - crate: viceroy
-            version: 0.2.4 # Note: workflow-level env vars can't be used in matrix definitions
+            version: *viceroy_version
```
Comment (JakeChampion) [inline .github/workflows/main.yml:96]:
```suggestion
            version: 0.2.14 # Note: workflow-level env vars can't be used in matrix definitions
```

**Review by elliottt — APPROVED** [review]
Good call!

**Pull request merged**